cmd/faInfo/testdata/expected.fa

```diff
@@ -0,0 +1,5 @@
+Name:fasta1	A:4	T:3	C:3	G:0	Gap:0	N:0
```

Comment:
Will it be easier to have a head line that is: "#Name A T C G Gap N" so that the columns are just numbers?  This will allow people to more easily use existing unix tools like "sort -k 4 -g" to analyze the file

Reply:
I agree and was considering this as I was making changes. I will change this.